src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-migrate/rgw-multisite-migrate.component.html

```diff
@@ -111,36 +111,18 @@
       </div>
       <div class="form-group row">
         <label class="cd-col-form-label required"
-               for="access_key"
-               i18n>S3 access key
-          <cd-helper>
-            <span>To see or copy your S3 access key, go to <b>Object Gateway > Users</b> and click on your user name. In <b>Keys</b>, click <b>Show</b>. View the access key by clicking Show and copy the key by clicking <b>Copy to Clipboard</b>.</span>
-          </cd-helper>
-        </label>
+               for="username"
+               i18n>Username</label>
```

Comment:
I'd recommend to add a helper here stating that this is to add a new user and not adding an existing to user of rgw.

Reply:
Thanks @nizamial09 , Updated